Polyhedron/demo/Polyhedron/Viewer.h

```diff
@@ -174,6 +175,7 @@ public Q_SLOTS:
   QString message;
   bool _displayMessage;
   QTimer messageTimer;
+  QMatrix4x4 projectionMatrix;
```

Comment:
@lrineau I suggest that this is done in another PR. Shall you open an issue for that?